bugwarrior/services/bz.py

```diff
@@ -150,9 +150,9 @@ def __init__(self, *args, **kw):
             force_rest_kwargs = {"force_rest": True}
 
         url = 'https://%s' % self.base_uri
-        api_key = self.config.get('api_key', default=None)
-        if api_key:
+        if self.config.get('api_key'):
             try:
+                api_key = self.get_password('api_key')
```

Comment:
Is there a reason this is going into the `try` block?

Reply:
No more than the next call to `get_password`, indeed. Good catch, thanks!